content/docs/command-reference/config.md

```diff
@@ -96,13 +96,27 @@ multiple projects or users, respectively.
 
 ## Configuration sections
 
-The following config sections are written by this command to the project config
-file (in `.dvc/config` by default), and they support the options below:
+The following config sections are written by this command to the appropriate
+config file (`.dvc/config` by default), supporting different config options
+within:
+
+- [`core`](#core) - main section with the general config options
```

Comment:
```suggestion
- [`core`](#core) - main section with the general config options.
```

Reply:
Not a full sentence 😬

Reply:
Nor do they complete a stem started before the bullet list